File: CustomerPortal/App/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Portal.Web
{
    public class Customer {

        public int _cusId;
        public String _firstName;
        public String _lastName;
        public String _email;
        public String _phoneNumber;
        public Boolean _status = false;
        

        public Customer() {
        }

        public Customer(int cusId, String firstName, String lastName, String email, String phoneNumber, Boolean status) {
             _cusId = cusId;
            _firstName = firstName;
            _lastName = lastName;
            _email = email;
            _phoneNumber = phoneNumber;
            _status = status;
            

        }

    }
}

File: WebApp/WebApp/app/customer.ts
class Customer {
    cusId : number;
    firstName :String ="";
    lastName: String = "";
    email :String ="";
    phoneNumber :String = "";
    status :boolean = true;
    
    constructor(cusId: number, firstName: String, lastName: String, email: String, phoneNumber: String, status: boolean) {
        this.cusId = cusId;
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.phoneNumber = phoneNumber;
        this.status = status;
    }

  
}
//app.service('customer', Customer);